docs/README.md

```diff
@@ -47,3 +47,12 @@ You can now start a local webserver via:
 bundle exec jekyll serve
 ```
 The website should now be accessible at [http://localhost:4000](http://localhost:4000)
+
+
+## Update the Readthedocs.io API documentation
+Use the following steps to update the public API documentation.
+
+1. Make your documentation changes and push them to the rtd-staging branch. This will rebuild the docs in the staging branch.
```

Comment:
Can you add a note that it's okay to force push to this branch to overwrite in previous changes?

Reply:
Done.